kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -3537,10 +3545,10 @@ messages:
       return;
    }
 
-   UserGet(what = $)
+   UserGet(what = $, number = $)
```

Comment:
This number parameter needs to be documented since it's not obvious what it does.

Reply:
Done with doc string.